src/components/ReleaseService/ReleasePlan/TriggerRelease/TriggerReleaseFormPage.tsx

```diff
@@ -36,7 +35,7 @@ export const TriggerReleaseFormPage: React.FC<Props> = ({ releasePlan, applicati
         workspace,
       });
       navigate(
-        `/application-pipeline/workspaces/${workspace}/applications/${applicationName}/releases/${newRelease.metadata?.name}`,
```

Comment:
This URL is correct; you don't need to change it, The Releases list page exists on the application details page. Once user triggers a release they should be redirected to the release details page.

Reply:
Sounds good!

Reply:
Link has been reverted: https://github.com/openshift/hac-dev/pull/990/commits/1c9351db37e465870a4b51c90c57602eebdaaede#diff-710fcbb03b424852fb2cdcb395bd2f4d74b11b91fd85cd0dbcec6bf381b7af63R39

Along with other changes needed for this revert, please see: https://github.com/openshift/hac-dev/pull/990/commits/1c9351db37e465870a4b51c90c57602eebdaaede#diff-710fcbb03b424852fb2cdcb395bd2f4d74b11b91fd85cd0dbcec6bf381b7af63